frontend/src/metabase/visualizations/components/settings/ChartSettingInputs.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import Input from "metabase/components/Input.jsx";
+
+// value is an array of strings. This component provides one input box per string
+export default function ChartSettingInputs({ value, onChange }) {
```

Comment:
If `value` is an array I'd use `values` instead to make things a little clearer.

Reply:
I wanted to do that but the key `value` is hardcoded into the chart settings code (I didn't get to pick it)

Reply:
unless there's some way to rename a value when I destructure?

for example in Clojure it would be `{values :value}`

How do I do that in JS?

Reply:
Yeah I believe `{values: value}` should work.

Reply:
Scratch that it's the other way around `value: values` is what you're looking for I think.